client/flutter/lib/main.dart

```diff
@@ -77,7 +95,7 @@ class _MyAppState extends State<MyApp> {
             textTheme: ButtonTextTheme.accent),
       ),
       home: Directionality(
-          child: HomePage(),
+          child: HomePage(analytics),
```

Comment:
Forgive my Flutter ignorance, but is there a better way to pass this throughout all of the different widgets in the app without having to pass it as an argument to every page? Assuming this is where Providers come in. If this is not a small change for this PR, then it should not be a blocker

Reply:
Yes. As state management wasn't set up, I passed it down manually. Will need refactoring later. After v1 probably - there's a PR adding provider already : https://github.com/WorldHealthOrganization/app/pull/633

Reply:
sounds good, not a blocker and resolving this comment